apps/web/src/app/modules/trial-initiation/trial-initiation.component.html

```diff
@@ -0,0 +1,59 @@
+<header class="tw-bg-background-alt2 tw-h-10">
+  <div class="tw-bg-background-alt2 tw-h-48 tw-absolute tw--skew-y-3 tw-w-full tw--z-10"></div>
+  <div class="tw-w-1/2 tw-bg-background-alt2 tw-h-40"></div>
+</header>
+
+<div class="tw-flex tw-justify-center tw--mt-12 tw-max-w-[1280px] tw-min-w-4xl tw-mx-auto tw-px-4">
```

Comment:
We can replace the header with this. It sets a height of 96, puts a negative margin of half that, and skews it. From what I can tell it's identical to the previous behavior.

```suggestion
<div class="tw-bg-background-alt2 tw-h-96 tw--mt-48 tw-absolute tw--skew-y-3 tw-w-full tw--z-10"></div>

<div class="tw-flex tw-justify-center tw-max-w-screen-xl tw-min-w-4xl tw-mx-auto tw-px-4">
```

Reply:
Just checked and this is good! 👍